tests/python_tests/samples/conftest.py

```diff
@@ -207,6 +213,36 @@ def download_test_content(request):
             os.remove(file_path)
 
 
+@pytest.fixture(scope="session")
+def generate_test_content(request):
+    """Download the test content from the given URL and return the file path."""
+    
+    test_data = request.config.cache.get("TEST_DATA", None)
+    
+    file_name, subdir = request.param if isinstance(request.param, tuple) else (request.param, "")
+    file_path = os.path.join(test_data, subdir, file_name)
+    if not os.path.exists(file_path):
+        os.makedirs(os.path.dirname(file_path), exist_ok=True)
+        logger.info(f"Generating test content {file_name}")
```

Comment:
```suggestion
        logger.info(f"Generating test content {file_path}")
```
similar to `download_test_content()`. That way the log message gives more info